### DISCUSSION TITLE
Custom Civilian Faction

### DISCUSSION THREAD

**Discussion by JWB502** [Vdauphin/HeartsAndMinds#1095] [Question and answer] [answered]
I was able to add custom factions for the new SOG Prairie DLC...however, the issue I am having is how do we get the civilians as well?

**Comment by Vdauphin** [comment, marked as answer]
Hello,
Take a look here:
https://github.com/Vdauphin/HeartsAndMinds/blob/master/%3DBTC%3Dco%4030_Hearts_and_Minds.Altis/core/def/mission.sqf#L355-L361

Cheers

**Reply by JWB502** [reply]
Cool! So Im assuming it would just be proper to use the call function first and then copy into the proper space just like adding BLU/OPF factions.

**Reply by Vdauphin** [reply]
Yes exactly, it is the same thing
You need to copy and paste to the civilian entry from the param.hpp

**Comment by Vdauphin** [comment]
I will update the faction list: https://github.com/Vdauphin/HeartsAndMinds/pull/1096

**Reply by JWB502** [reply]
amazing as always 🚀
One day, I'll make a video and show you!
